frontend/src/components/controls/index.js

```diff
@@ -19,11 +19,26 @@ const playButton = (cb, playbackState, disabled) => (
   </Button>
 )
 
+const pauseButton = (cb, playbackState, disabled) => (
```

Comment:
Not functionally important but you might be at a point with the stop/play/pause buttons that they could be dryed up a bit e.g.

```suggestion
const button = (action, onClick, playbackState, disabled) => (
  <Button
    onClick={ onClick }
    animated='vertical'
    disabled={(pausedOrStopped(playbackState) || disabled)}
    active={paused(playbackState)}
    className=`jb-${action}-button`
  >
    <Button.Content hidden>{ action }</Button.Content>
    <Button.Content visible>
      <Icon name={action} />
    </Button.Content>
  </Button>
)

```

Reply:
You're right. I actually didn't in this case as I'm not sure how the UI is going to progress, so did the least amount of thinking on it.